sql/payday.sql

```diff
@@ -76,12 +76,27 @@ CREATE INDEX ON payday_payment_instructions (participant);
 CREATE INDEX ON payday_payment_instructions (team);
 ALTER TABLE payday_payment_instructions ADD COLUMN is_funded boolean;
 
+UPDATE payday_payment_instructions ppi
+   SET due = s.due
+  FROM (SELECT participant, team, SUM(due) AS due
```

Comment:
Why are we summing `due` here? My understanding was that `due` in `current_payment_instructions` would always be cumulative.